contracts/PetPark.sol

```diff
@@ -1,7 +1,90 @@
-//SPDX-License-Identifier: Unlicense
+// SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
 
 contract PetPark {
 
-}
\ No newline at end of file
+    address owner; 
+
+    enum AnimalType {
+        NONE,
+        Fish,
+        Cat,
+        Dog,
+        Rabbit,
+        Parrot
+    }
+
+    enum Gender {
+        Male,
+        Female
+    }
+
+    struct Borrower {
+        AnimalType animal;
+        bool isBorrowing;
```

Comment:
Since storage in solidity is expensive, can't `isBorrowing` be checked with stored animal being `NONE`

Reply:
yes sir... It does make sense to use that value